### DISCUSSION TITLE
Updating Character Settings - JiTouch

### DISCUSSION THREAD

**Discussion by dhan45** [JitouchApp/Jitouch#15] [Ideas]
Hello! I have a request for the _characters_ **tab** setting! For the distance between the index-ring, could we have the distance be minimized even more? IDK how exactly it's programmed, but the new MacBooks have much bigger trackpads, so I was wondering for the next update if it's possible to allow the distance between index-ring to be optimized to go even closer if wanted though that slide mechanism? I still feel like the distance is quite too far. That would be amazing! 

ALSO LOOVE THIS APP TO DEATH! Will be making a YouTube video about it! I hope you guys check it out. Will post link when it's up! Thanks :)

My email: davidhan1995@gmail.com

Best,
David

**Comment by aaronkollasch** [comment]
Hi David!

Yeah, I recently tried using characters and have the distance slider set all the way down and it's still too big I think. The range should be enough to make the distance "too small" if desired, so I'll look into it.

**Reply by aaronkollasch** [reply]
The smaller distance (#17) has been released in version 2.80. The auto-set distance feature works properly for me now, and I find the character gestures are a lot less awkward.
